src/NuGet.Core/NuGet.Configuration/Utility/FileSystemUtility.cs

```diff
@@ -29,7 +29,7 @@ internal static bool DoesFileExistIn(string root, string file)
             return File.Exists(Path.Combine(root, file));
         }
 
-        internal static IEnumerable<string> GetFilesRelativeToRoot(string root, string path, string[] filters = null, SearchOption searchOption = SearchOption.TopDirectoryOnly)
+        internal static IEnumerable<string> GetFilesRelativeToRoot(string root, string path = "", string[] filters = null, SearchOption searchOption = SearchOption.TopDirectoryOnly)
         {
```

Comment:
Please use string.Empty instead of "".

Reply:
@erdembayar, I don't believe you can use string.Empty as a default value because it's not a compile-time constant.

Reply:
> @erdembayar, I don't believe you can use string.Empty as a default value because it's not a compile-time constant.

You're right.

Reply:
Yep, that's the exact reason as @debonte said :) 
Gotten me so many times before :)